docs/Permissions/Onchain-Permissioning/Using-Onchain-Permissioning.md

```diff
@@ -0,0 +1,207 @@
+description: Setting up and using onchain Permissioning
+<!--- END of page meta data -->
+
+# Using Onchain Permissioning 
+
+To start a network with onchain permissioning: 
+
+1. [Install pre-requisites](#pre-requisites) 
+1. [Add the ingress contracts to the genesis file](#add-ingress-contracts-to-genesis-file) 
+1. For the first node: 
+    1. [Set environment variables](#set-environment-variables) 
+    2. [Start node with onchain permissioning and the JSON-RPC HTTP service enabled](#onchain-permissioning-command-line-options) 
+    3. [Clone the permissioning contracts repository and install dependencies](#clone-contracts-and-install-dependencies) 
+    4. [Deploy the permissioning contracts](#deploy-contracts) 
+    5. [Start the Permissioning Management Dapp](#start-the-permissioning-management-dapp) 
+    6. [Add the first node to the nodes whitelist](#update-nodes-whitelist)
+    7. Add subsequent [nodes](#update-nodes-whitelist) and [accounts](#update-accounts-whitelists) to the whitelists 
+
+1. Start subsequent nodes with [onchain permssioning enabled](#onchain-permissioning-command-line-options).  
+If nodes are going to maintain whitelists, also enable the [JSON-RPC HTTP service](#onchain-permissioning-command-line-options). 
+1. For nodes that are going to maintain whitelists (that is, nodes with an admin account): 
+    1. [Clone the permissioning contracts repository and install dependencies](#clone-contracts-and-install-dependencies)
+    2. [Start the Permissioning Management Dapp](#start-the-permissioning-management-dapp)
+
+## Pre-requisites 
+
+For the first node and nodes with an admin account: 
```

Comment:
```suggestion
For the first node, and other nodes with an admin account: 
```

Reply:
There are a couple of things here that I don't know if we should care (might be nitpicking)

1. The concept "nodes with an admin account" doesn't make sense to me... The account doesn't belong to one node.
2. You don't need to run the scripts and the dapp in the same server that you are running the node.

I think we need to make clear that this is the way to bootstrap the network. Hence why you need one dapp running.

Reply:
I think we can remove the whole `If nodes are going to maintain whitelists` part.

Reply:
Have updated throughout to make clearer.